partialWaveFit/fitResult.cc

```diff
@@ -998,3 +1000,149 @@ fitResult::intensityErr(const string& waveNamePattern) const
 	const TMatrixT<double> intensityCov = jacobian * prodAmpCovJT;              //  1 x  1 matrix
 	return (double)normNmbEvents() * sqrt(intensityCov[0][0]);
 }
+
+
+void fitResult::stripCovMatrix() {
+	_fitParCovMatrix.ResizeTo(0,0);
+	_fitParCovMatrixIndices.clear();
+	_hasHessian = false;
+	_covMatrixValid = false;
+}
+
+
+void fitResult::stripIntegralMatrices() {
+	_normIntegral.resizeTo(0,0);
+	_acceptedNormIntegral.resizeTo(0,0);
+	_phaseSpaceIntegral.clear();
+	_normIntIndexMap.clear();
+}
+
+
+std::map<rpwa::multibinBoundariesType, std::list<rpwa::fitResult> > rpwa::getFitResultsFromFilesInMultibins(
```

Comment:
why return a `list` of fit results, not a `vector`? Internally the usage of a `list` reduces the number of copies of fit result objects created, and simplifies a number of things. But for external use it does not allow direct access to specific elements. Converting to a python object requires an extra step. Are the internal advantages really worth the external hassles?

Reply:
The main reason was, that the sorting should be more efficient, as the fit results are big objects and also the iterators I use above stay valid after inserting a new element, which is not the case for a vector (see https://stackoverflow.com/questions/6438086/iterator-invalidation-rules).

Internally, I would stick to the `list` type, anyway.
The only use case I see so far is the one via the Python interface.
So I do not think it is rally advantageous to make here a conversion to `vector`, because we do the conversion than in the Python interface.